Referenced by commit f24854005d8, Fix T94708: negative reference count error with Python API callbacks
